### DISCUSSION TITLE
Conquista Especiais/Missões no Aplicatiovo (Próximas versões)

### DISCUSSION THREAD

**Discussion by ikidoncc** [diego3g/rsxp-2023#13] [Ideas]
Primeira ideia que o Diego deu em Live + Complemento meu.
- Conhecer e conversa com alguns palestrantes e obter o QRCode deles (Conquista de Celebridades)
- Conhecer e obter todos os QRCode dos palestrantes (Conquista de Amante do Palco)
- Conhecer os membros da Rocketseat presente no Evento e obter o QRCode deles (Conquista de Rocket Flow)
- Ter um album com todas essas conquistas

Segunda ideia, conquistas especiais dentro do Evento, só que escondidas.
- Achar QRCodes escondidos durante o evento (Conquista de Explorador da RS/XP).
- Decifrar enigmas para liberar QRCode (Conquista de E=mc²).
- Resolver alguns códigos escondidos (Conquista de Em busca do Código perdido).
- Conhecer X pessoas durante o evento (Conquista de  Conexão Nível Rocket).

Creio que o ambiente ficaria mais divertido e gamificado, do jeito que a galera ama, muita diversão, alegria, brincadeira e código

**Comment by ruymon** [comment]
Buenas! 

Dê uma olhada nessas threads aqui! #3 #5 #8

Acredito que são ideias semelhantes que podem ser unificadas:)
